modules/View.js

```diff
@@ -31,6 +31,9 @@
       if (name === 'change' || name.indexOf('change:') === 0) {
         View.prototype.trigger.apply(this.view, arguments);
       }
+      if (name.indexOf('change:hide:') === 0) {
```

Comment:
this makes me cringe for some reason, but I think its ok.

Reply:
it's low risk, so maybe that makes seeing if the potential is worth it?

Reply:
I think the cringe was because it calls render directly.  I'm used to pushing render to be event triggered to prevent unnecessary renders.  But this is event triggering so... it was only a hesitation not a reason not to.